tasks/main.yml

```diff
@@ -21,6 +21,15 @@
     state: latest
   tags: [influxdb]
 
+- name: Install system dependencies
+  apt:
+    name:
+      - curl
+      - rsyslog
```

Comment:
why `rsyslog` here?

Reply:
A following task updates rsyslog logrotate configuration. that configuration file does not exist if rsyslog is not installed and thus the task will fail. 

https://github.com/openwisp/ansible-ow-influxdb/blob/3445e0230f80411f0d06df5940244859d6528753/tasks/main.yml#L92-L104